_data/xenapi.json

```diff
@@ -11847,8 +11855,7 @@
 "lifecycle": [ 
   { "transition": "published",
     "release": "cowley",
-    "description": "A tunnel for network traffic" } ],
-"tag": "networking" }, { "name": "PCI",
```

Comment:
This patch currently reverts a few recent changes I made. Using xapi-project/xen-api#2185 to generate xenapi.json will fix this.